VideoWeb/VideoWeb/ClientApp/src/app/services/conference/models/virtual-meeting-room.model.ts

```diff
@@ -0,0 +1,25 @@
+import { ParticipantModel } from 'src/app/shared/models/participant';
+import { RoomSummaryResponse } from '../../clients/api-client';
+import { PexipDisplayNameModel } from './pexip-display-name.model';
+
+export class VirtualMeetingRoomModel {
+    public pexipDisplayName: PexipDisplayNameModel | null;
```

Comment:
instead of casting null type, can we use this instead? 
`public pexipDisplayName?: PexipDisplayNameModel`

Reply:
I think specifying that it can be null is more explicit. Using the ? you are saying it is optional, where as it shouldn't be it should be either null - the participant doesn't have one or a PexipDisplayNameModel.